docs/adguard-for-linux/settings.md

```diff
@@ -51,7 +90,30 @@ Use the `filters` command to configure AdGuard for Linux. Subcommands:
 
 - `list`: List installed filters
     - `--all`: View all filters
+
+    ![Filter list *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/filter-list.png)
+
 - `install`: Install a filter. Enter the URL of the filter you want to install
 - `enable`: Enable a filter. Enter the name or ID of the filter
+
+    ![Enable filters *border](https://cdn.adtidy.org/content/Kb/ad_blocker/linux/built-in-filters.png)
+
 - `disable`: Disable a filter. Enter the name or ID of the filter
 - `update`: Update filters
+
+## Changing the proxy server listen address in manual proxy mode
+
+By default, the proxy server listens on `127.0.0.1` — the address of the loopback network interface.
+There are two ways to make the proxy server listen on a different interface:
+
+1. Run `adguard-cli config set listen_address <address>` where `<address>` is the address to listen on.
```

Comment:
All steps are `1`

Reply:
they should be. @puglieri please it the way it was